NitroxLauncher/BepInEx/BepInEx/config/BepInEx.cfg

```diff
@@ -0,0 +1,16 @@
+[Preloader.Entrypoint]
+
+## The local filename of the assembly to target.
+# Setting type: String
+# Default value: UnityEngine.CoreModule.dll
+Assembly = Assembly-CSharp.dll
+
+## The name of the type in the entrypoint assembly to search for the entrypoint method.
+# Setting type: String
+# Default value: Application
+Type = SystemsSpawner
```

Comment:
Why the change? What's the difference between `SystemsSpawner` and `GameInput`?

Reply:
2 fold.
1. SystemsSpawner is the VERY first monobehavior to run in the game. It is what actually Creates the gameobject with the GameInput monobehavior on it.
2. Currently this is the location QMM will be using and the config is one file that both will use and doesn't really need to be switched out.